cobra/core/model.py

```diff
@@ -149,6 +154,18 @@ def solver(self, value):
             return
         self._solver = interface.Model.clone(self._solver)
 
+    @property
+    def tolerance(self):
+        return self._tolerance
+
+    @tolerance.setter
+    def tolerance(self, value):
+        solver_config = self._solver.configuration
```

Comment:
Would this work?
```suggestion
        tolerances = self._solver.configuration.tolerances
```

Reply:
Would you want that to be a dictionary?

Reply:
No, it was just an idea to shorten the remaining lines.

Reply:
It should work. I will push the change. But, I would prefer `solver_tolerances` instead of `tolerances` so that it's a bit more explicit.

Reply:
Perfect.